### DISCUSSION TITLE
Type instantiation is excessively deep and possibly infinite in nuxt.config.ts

### DISCUSSION THREAD

**Discussion by SeppeP** [nuxt/nuxt#17638] [Questions] [answered]
I just updated my Nuxt3 project to the release candidate version of Nuxt3. The part where i configure vite in nuxt.config.ts now gives me a typescript error i'm not familiar with: 'Type instantiation is excessively deep and possibly infinite'.

<img width="634" alt="image" src="https://user-images.githubusercontent.com/56549914/166224181-34043fb0-a1d7-4d96-99da-2f85029f9a36.png">

Can anyone explain what this means or how to solve this?

**Comment by kmuip** [comment]
I have the same problem, in my case it's the strapi plugin that causes this problem in the nuxt.config.ts file.

![image](https://user-images.githubusercontent.com/73472029/166260221-9ce37b36-5aa3-468d-b66f-9699c589d1dd.png)

Btw: According to the [release notes](https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.1) of the nuxt3 rc, you should import defineNuxtConfig from nuxt :)

**Reply by SeppeP** [reply]
importing from nuxt3 was indeed a silly mistake on my part. Unfortunately fixing that doesn't solve my problem.

**Reply by kmuip** [reply]
Was just an fyi :)

**Comment by danielroe** [comment, marked as answer]
This issue should be resolved in the next RC release (or already, if you are on the edge channel).

**Reply by SeppeP** [reply]
Okay, thanks!

**Reply by kmuip** [reply]
Awesome!

**Reply by florian-lefebvre** [reply]
Hey @danielroe, upgrading to RC 2 didn't solve the issue on my side (error is triggered for the Tailwind CSS module). Here is the concerned repo if you want to have a look: https://github.com/florian-lefebvre/nuxt3-starter-medusa

**Reply by danielroe** [reply]
@florian-lefebvre I don't get it on that repo when I clear your lockfile and reinstall. Can you check and if it persists, open a new issue?

**Reply by florian-lefebvre** [reply]
Sure
